setup.py

```diff
@@ -23,6 +23,7 @@
         'tornado>=5.1,<6.0.0',
         'regex>=2017.9.23',
         'PyYAML>=4.2b4,<5.0',
+        'aiohttp>=3.5.4,<4.0',
```

Comment:
also add to `requirements_dev.txt`

Reply:
Does requirements_dev need to include the base synapse requirements?  it currently doesn't.

Reply:
`aiohttp` isn't a base requirement though; it's only used inside of test code. In that case, it should go down in the `extras_require` area and added to 
 `requirements_dev.txt`.  The `requirements_dev.txt` also pulls in `requirements.txt` when it is resolved.

This is really no different than something like pytest, which is required in order to execute individual unit tests (where the built-in unittest package won't work because of how unittest does its test loading).

Reply:
ah, i see.  so, i added it where i did because i'll also be implementing some async http client helper stuff in (for example) storm that will be using it directly.  Just preping for the very near future. :)

Reply:
( in a subsequent PR )